src/content/en/showcase/2017/within.md

```diff
@@ -0,0 +1,386 @@
+
+{# wf_published_on: 2017-02-02T18:00:00.000Z #}
+{# wf_updated_on: 2017-02-02T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/within/featured.png #}
+{# wf_featured_snippet: Within is a platform for storytelling in Virtual Reality and is available everywhere VR is. This includes the web. Leveraging WebVR, viewers are able to go to the website, click a link, and immediately watch Within’s films in immersive VR - including high end head-mounted displays. During the development process the team discovered rendering text is difficult in this new environment, and they created an example using shaders to make it a smoother process. #}
+{# wf_tags: webvr,casestudy #}
+
+
+# Rendering Text in WebVR {: .page-title }
+
+*By David Lyons, Lead WebVR Developer, Within*
+
+<img src="/web/showcase/2017/images/within/0.png" class="attempt-right">
+
+<a class="button button-primary" href="https://vr.with.in/">
+  View the site
+</a>
+
+## Introduction
+
+Within ([http://with.in/](http://with.in/)) is a platform for storytelling in
+Virtual Reality. So when the team heard about WebVR in 2015 we were immediately
+interested in its potential. Today that interest manifests into a unique
+subdomain of our Web Platform, [https://vr.with.in/](https://vr.with.in/). Anyone
+with a VR-enabled browser can go to the site, click a button and throw on a
+headset to be immersed in our portfolio of VR films.
+
+*Today that includes but is not limited to Chrome on Daydream View. For
+information on your device and head-mounted display check out
+[https://webvr.info/](https://webvr.info/).*
+
+Like other Virtual Reality specific rendering environments, the web
+predominantly relies on a three-dimensional representation of a scene. This
+scene has a camera, your perspective, and any number of objects. To help manage
+this scene, camera, and objects we use a library called
+[Three.js](http://threejs.org/) which leverages the `<canvas>` element to throw
+rendering onto your computer's GPU. There are many useful Three.js add-ons to
+make your scene viewable in WebVR. The main two are
+[THREE.VREffect](https://github.com/mrdoob/three.js/blob/dev/examples/js/effects/VREffect.js)
+for creating a viewport for each eye and
+[THREE.VRControls](https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/VRControls.js)
+for translating the perspective (e.g the rotation and position of the
+head-mounted display) convincingly into your scene. There are many examples of
+how to implement this. Check out the
+[Three.js WebVR examples](https://threejs.org/examples/?q=webvr)
+for ways to get started.
+
+As we got further into our exploration of WebVR we ran into an issue. If we look
+at the contents of the web, text is an integral part of it. While the majority
+of our content is video based, if you go to the
+[Within site](http://with.in/#films) text surrounds the content;
+user-interface, additional information about a film or related films are all
```

Comment:
This sounds weird to me. Perhaps: 'the user interface and additional information. Also 'user interface', not 'user-interface'.